build_calicoctl/create_binary.sh

```diff
@@ -1,14 +1,32 @@
 #!/bin/bash
 
 set -e
+set -x
 
 docker build -t calico-build .
 
 cd ..
 mkdir -p `pwd`/dist
 chmod 777 `pwd`/dist
-docker run -u user -v `pwd`/:/code/calico -v `pwd`/dist:/code/dist calico-build pyinstaller calico/calicoctl.py -a -F -s --clean
-docker run -u user -v `pwd`/dist:/code/dist calico-build bash -c 'docopt-completion --manual-bash dist/calicoctl && mv calicoctl.sh dist'
+
+if [[ $CIRCLE_TEST_REPORTS ]]; then
```

Comment:
Hmmm, I don't like this approach. The circle specific parts should go in the circle.yaml